apache-cassandra-mixin/config.libsonnet

```diff
@@ -1,5 +1,9 @@
 {
   _config+:: {
+    enableMultiCluster: true,
```

Comment:
This will probably have inclinations of the integration, just a heads up by default we will have a cluster label when importing and using this mixin when its updated.

Reply:
@schmikei good catch. this was never meant to be pushed as "true". i switched it while testing, but forgot to switch it back to "false". i'll change.